test/test_automl/test_automl.py

```diff
@@ -307,10 +321,23 @@ def test_automl_outputs(self):
         self._tearDown(backend_api.output_directory)
 
     def test_do_dummy_prediction(self):
-        for name in ['401_bac', '31_bac', 'adult', 'cadata']:
```

Comment:
Could you please check whether we can delete those dataset files from the repository or whether they are still needed?